### PR TITLE
fix(clerk-js): Revert user settings work

### PR DESCRIPTION
**Pull request by igneel64**
## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/types`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Revert `User Settings` feature.

### CODE REVIEW

**Comment by SokratisVidros** [comment]
@igneel64 Looking good. Please squash and merge it so that we can easily put it back next week. @nikosdouvlis Can you please verify it as well?

**Comment by SokratisVidros** [comment]
@igneel64 Looking good. Please squash and merge it so that we can easily put it back next week. @nikosdouvlis Can you please verify it as well?

File: packages/clerk-js/src/ui/signIn/SignInStart.tsx [hunk 1]
```diff
@@ -36,28 +46,60 @@ export function _SignInStart(): JSX.Element {
   const instantPassword = useFieldState('password', '');
   const [error, setError] = React.useState<string | undefined>();
 
-  const standardFormAttributes = userSettings.enabledFirstFactorIdentifiers;
-  const web3FirstFactors = userSettings.web3FirstFactors;
-  const socialProviderStrategies = userSettings.socialProviderStrategies;
-  const passwordBasedInstance = userSettings.instanceIsPasswordBased;
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/signIn/SignInStart.tsx:42]: This PR seems to remove the instant password fix. Should we wait for UserSettings or should we implement a fix for the AuthConfig version? cc @SokratisVidros

Comment (SokratisVidros) [reply packages/clerk-js/src/ui/signIn/SignInStart.tsx:42]: We should keep the fix despite the revert

**Review by nikosdouvlis — APPROVED** [review]

**Pull request merged**